core/src/main/resources/hudson/model/queue/CauseOfBlockage/BecauseLabelIsBusy/summary_de.properties

```diff
@@ -20,4 +20,3 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 
-description=Alle Knoten des Labels ''{0}'' sind besch\u00E4ftigt
```

Comment:
Something's weird here and in the next few files, will need to investigate. Perhaps these are obsolete?